### DISCUSSION TITLE
How long after disabling AdGuard Home should blocked queries start resolving?

### DISCUSSION THREAD

**Discussion by darrendavid** [AdguardTeam/AdGuardHome#6777] [Q&A]
When I find a blocked URL that I need to access, I jump over to the AdGuard Home dashboard and disable AdGuard, however, it takes about 45s to a minute before those queries are no longer blocked and resolve properly. Seems to be the same with adding a new custom query to allow a URL. Is there a way to speed this up?

**Comment by whyisthisbroken** [comment]
You find it under "Settings \ DNS Settings"
There is "Blocked Response TTL" - should be set at default to "10".

**Reply by darrendavid** [reply]
Thanks for the tip - It is set to 10, but that's not the behavior I'm seeing. I've tried flushing my DNS cache on my local machine after disabling the blocker, but that's doesn't seem to speed things up.

**Reply by Dynamic5912** [reply]
Disconnect and reconnect to the WiFi/network again. It's probably DNS cached locally on the device.
